tests/fixtures/docker_cluster_fixtures.py

```diff
@@ -407,6 +408,8 @@ def docker_cluster_pk_ssh(request, test_org_rns_folder):
     if not detached:
         cleanup()
 
+    delete_test_resource(test_org_rns_folder)
+
```

Comment:
maybe just add this to the `test_org_rns_folder` fixture? instead of the docker cluster fixture?